### PR TITLE
Fix CI/CD issue caused by action server's performance

### PR DESCRIPTION
**Pull request by ACMoretxj**

### CODE REVIEW

File: deploy/default-azuredeploy.json [hunk 1]
```diff
@@ -34,17 +34,11 @@
     },
     "imageTag": {
       "type": "string",
-      "allowedValues": [
-        "v1.0.0",
-        "v2.0.0",
-        "v2.1.0"
-      ],
-      "defaultValue": "v2.1.0"
+      "defaultValue": "healthplatformregistry.azurecr.io/fhirconverter:v2.1.0"
```
Comment (BoyaWu10) [inline deploy/default-azuredeploy.json:37]: If we are storing the complete one here, maybe it does not belong to `imageTag`?

File: test/e2e-test/rule-test/util/test-rule-functions.spec.js [hunk 2]
```diff
@@ -240,7 +240,7 @@ describe('testRule', function () {
         const reqJson = null;
         const resJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../test-samples/FHIR-R4/sample3-right.json')));
         assert.ok(testRules.officialValidator(reqJson, resJson).valid);
-    }).timeout(60000);
+    }).timeout(180 * 1000);
```
Comment (BoyaWu10) [inline test/e2e-test/rule-test/util/test-rule-functions.spec.js:243]: Does the issue caused by the timeout settings?

Comment (ACMoretxj) [reply test/e2e-test/rule-test/util/test-rule-functions.spec.js:243]: Yes, the performance of github action server is not that stable, so the time usage of validating phase will fluctuate.
I change the timeout limit to a relatively big number to eliminate the influence of performance fluctuation.

File: deploy/default-azuredeploy.json (hunk 1 above)
Comment (ACMoretxj) [reply deploy/default-azuredeploy.json:37]: Absolutely, the `imageTag` is not a good name,  I'll change it to `imagePath`.

File: test/e2e-test/rule-test/util/test-rule-functions.spec.js (hunk 2 above)
Comment (BoyaWu10) [reply test/e2e-test/rule-test/util/test-rule-functions.spec.js:243]: Got it. Thanks!

**Review by BoyaWu10 — APPROVED** [review]

**Review by qiwjin — APPROVED** [review]

**Pull request merged**